polaris-for-vscode/README.md

```diff
@@ -1,38 +1,26 @@
-# `polaris-for-vscode` Extension
+# Polaris for VSCode
 
-:warning: This extension is still an experimental prototype
+Official VSCode extension for building with the Shopify [Polaris Design System](https://polaris.shopify.com/).
 
-VSCode features to support developing with the Polaris Design System.
+![Demo of Polaris for figma autocompleting token code](https://github.com/Shopify/polaris/blob/main/polaris-for-vscode/docs/polaris-for-vscode-preview.gif?raw=true)
```

Comment:
I can't get the extension marketplace to resolve a relative URL so we will need to merge in this PR first before it shows up correctly in the packaged extension

Reply:
If you check out the [shopify theme check](https://github.com/Shopify/theme-check-vscode/blob/main/README.md?plain=1#L3) extension they also had to use this method